src/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -165,6 +165,14 @@ public static double Tanh(double value)
             return RuntimeImports.tanh(value);
         }
 
+        internal static double CopySign(double x, double y)
+        {
```

Comment:
I definitely agree that it would be nice to remove this ifdef complete - it does not seem to be warranted.

I do not actually understand the purposes of _copysign here. As far as I can tell, the result has the right sign already and the odd-case of negative zero was handled by the special case at the top of the function. Could you please try removing it and see whether there are any tests failing (on either .NET Native for UWP or CoreCLR) ?